Exiled.API/Features/Player.cs

```diff
@@ -1912,6 +1912,21 @@ public void Heal(float amount, bool overrideMaxHealth = false)
                 Health += amount;
         }
 
+        /// <summary>
+        /// Uses an item for giving its effect.
+        /// </summary>
+        /// <param name="usableItem">The item to be used.</param>
+        public void UseItem(ItemType usableItem)
+        {
+            if (Item.Create(usableItem, this) is not Usable item)
+                throw new Exception($"The provided item [{usableItem}] is not a usable item.");
```

Comment:
Maybe throw **new InvalidOperationException**? Also "The provided item [{usableItem}] is not a**n** usalbe item."